<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(60, 59, 55);
	color: rgb(186, 189, 182);
	font: 11pt &quot;Ubuntu&quot;;
}
QPushButton
{
	alternate-background-color: rgb(60, 59, 55);
	background-color: rgb(79, 75, 75);
}
QLabel#lbl_content_img, QLabel#lbl_output_img, QLabel#lbl_progress_img
{
	font: 57 42pt &quot;Ubuntu&quot;;
	color: rgb(233, 185, 110);
	background-color: rgb(85, 87, 83);
}

QSlider
{
	selection-background-color: rgb(233, 185, 110);
}

QRadioButton
{
	color: rgb(245, 226, 143);
}
QProgressBar
{
	font: 57 italic 13pt &quot;Ubuntu&quot;;
	color: rgb(206, 92, 0);
	selection-background-color: rgb(233, 185, 110);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="btns_layout">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_select_content">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_get_foreground">
       <property name="text">
        <string>前景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_foreground_view">
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="output_layout">
     <item>
      <widget class="QLabel" name="lbl_output_img">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="content_layout">
     <item>
      <widget class="QLabel" name="lbl_content_img">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="btns_layout_2">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_args">
       <property name="text">
        <string>参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_style_transfer">
       <property name="text">
        <string>生成</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_progress">
       <property name="text">
        <string>进度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>1021</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_style">
     <attribute name="title">
      <string>选择风格/框架</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1021</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="style_layout">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_style_1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_style_1">
          <property name="geometry">
           <rect>
            <x>18</x>
            <y>20</y>
            <width>251</width>
            <height>181</height>
           </rect>
          </property>
          <property name="text">
           <string>style_img1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_style_1">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>240</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_style_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_style_2">
          <property name="geometry">
           <rect>
            <x>18</x>
            <y>20</y>
            <width>251</width>
            <height>181</height>
           </rect>
          </property>
          <property name="text">
           <string>style_img2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_style_2">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>240</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_style_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_style_3">
          <property name="geometry">
           <rect>
            <x>18</x>
            <y>20</y>
            <width>251</width>
            <height>181</height>
           </rect>
          </property>
          <property name="text">
           <string>style_img3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_style_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>240</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_args">
     <attribute name="title">
      <string>参数</string>
     </attribute>
     <widget class="QSlider" name="slider_iterations">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>迭代次数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_iterations">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>内容权重：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_content_weight">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>180</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1e3</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_content_weight">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>风格权重：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_style_weight">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>280</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1e-2</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_style_weight">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_progress">
     <attribute name="title">
      <string>生成进度</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>风格转换：</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_transfer">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>821</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_progress_img">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>479</width>
        <height>299</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
